.gitignore

```diff
@@ -1,4 +1,4 @@
-
+hypershift
```

Comment:
should the binary not go into bin/hypershift? not sure we need this ignore

Reply:
removed